website/docs/r/datasource.html.markdown

```diff
@@ -18,6 +18,13 @@ resource "ns1_datasource" "example" {
   sourcetype = "nsone_v1"
 }
 ```
+OR
```

Comment:
Why not add this to the example above with a unique resource name?

Reply:
fixed

Reply:
Still appears to be a seperate code block, any reason we can't remove the `OR` and combine both resources into a single code block, as in other examples?